docs/installation-docker.rst

```diff
@@ -0,0 +1,81 @@
+=======================
+Installation via Docker
+=======================
+
+Kuma has experimental support for `Docker`_ as an alternative to the
+Vagrant set up.
+
+.. _Docker: https://www.docker.com/
+
+Docker setup
+============
+
+#. Ensure you have cloned the kuma Git repository and it is your current
+   working directory::
+
+        git clone git@github.com:mozilla/kuma.git
+        cd kuma
+
+#. Install the `Docker Toolbox`_. This will install the Docker engine,
+   along with Docker Compose and Docker Machine.
+
+   .. _Docker Toolbox: https://www.docker.com/products/docker-toolbox
+
+#. Create the virtual machine that Docker will run within::
+
+        docker-machine create --driver virtualbox --virtualbox-memory 2048 kuma
+
+#. Configure the environment variables::
+
+        eval "$(docker-machine env kuma)"
+        export COMPOSE_FILE=`pwd`/docker/docker-compose.yml
+        export COMPOSE_PROJECT_NAME=kuma  # String prepended to every container.
+
+#. Add the Docker VM to your hosts::
+
+        sudo sh -c "echo $(docker-machine ip kuma 2>/dev/null)  kuma.dev >> /etc/hosts"
```

Comment:
There is no need to use `sudo sh` for this. Docker has a `--add-host` option: https://docs.docker.com/engine/reference/run/#managing-etc-hosts Now to find out how to make this work in our workflow.

Also, `kuma.dev` doesn't match the SSL cert we're using, which we have to while we're on Persona at least and I'd make the argument that using SSL in development prevents us from making dev/prod parity mistakes. So let's keep using `developer-local.allizom.org` for it and make sure the embedded attachment files are correctly served like in stage/prod by additionally using `mdn-local.mozillademos.org`, too.